src/plays/image-gallery-using-unsplash-api/styles.css

```diff
@@ -0,0 +1,112 @@
+/* enter stlyes here */
+.body {
```

Comment:
Have made changes here. Earlier it was *{ padding: 0;
  margin: 0;
  font-family: 'IBM Plex Sans', sans-serif;
  background-color: whitesmoke;
} .

Now it is under a className body.

Reply:
The class name should prefix all the styles in the CSS file.